README.md

```diff
@@ -63,16 +219,92 @@ You can find an example project [here](https://github.com/uploadcare/pyuploadcar
             'data-cdn-base': 'https://cdn.super-candidates.com',
             'data-image-shrink': '1024x1024',
         }))
-
 ```
 
-![](https://ucarecdn.com/f0894ef2-352e-406a-8279-737dd6e1f10c/-/resize/800/manual_crop.png)
+![](https://ucarecdn.com/f0894ef2-352e-406a-8279-737dd6e1f10c/-/resize/800/josi.png)
 
-## Documentation
+## Testing
+
+To run tests using [Github Actions](https://github.com/uploadcare/pyuploadcare/actions) workflows, but locally, install the [act](https://github.com/nektos/act) utility, and then run it:
+
+```make test_with_github_actions```
+
+This runs the full suite of tests across Python and Django versions.
+
+<!--
```

Comment:
Work in progress?

Reply:
Or it shouldn't belong here. IDK.. Maybe it's better as a separate section in actual documentation.